library.json

```
@@ -1,27 +1,48 @@
 {
   "$schema": "https://raw.githubusercontent.com/platformio/platformio-core/develop/platformio/assets/schema/library.json",
-  "name": "LibraryTemplate",
+  "name": "Min-heap",
```

As suggested [here](https://docs.platformio.org/en/latest/manifests/library-json/fields/name.html) for simplicity, consistency, and compatibility the name of the library should be PascalCase.
```suggestion
  "name": "MinHeap",
```